src/baton-put.c

```diff
@@ -60,10 +62,11 @@ int main(int argc, char *argv[]) {
             {"unbuffered",    no_argument, &unbuffered_flag,    1},
             {"unsafe",        no_argument, &unsafe_flag,        1},
             {"verbose",       no_argument, &verbose_flag,       1},
+	    {"verify",        no_argument, &verify_flag,        1},
             {"version",       no_argument, &version_flag,       1},
             {"wlock",         no_argument, &wlock_flag,         1},
             // Indexed options
-            {"connect-time", required_argument, NULL, 'c'},
+            {"connect-time",  required_argument, NULL, 'c'},
```

Comment:
Should this be included in the third argument of getopt_long_only() on line 76 as "c:b:f:"?

Reply:
Yes, that's a bug.